unicode-transforms.cabal

```diff
@@ -23,9 +23,14 @@ tested-with:    GHC==7.10.3
               , GHC==8.10.1
 build-type:     Simple
 extra-source-files:
+    .gitignore
+    .travis.yml
+    appveyor.yml
```

Comment:
Is it intended? I do not see much point in uploading these configs onto Haskell, but I'm fine that your policy about this is different.

Reply:
Some people argue that you should package everything to reproduce the dev environment. I have no strong opinion on it, either way is fine. I would avoid packaging them if these were big files.

Reply:
OK, makes sense.